backend/entityservice/views/project.py

```diff
@@ -217,7 +219,11 @@ def project_clks_post(project_id):
                 #       consume the stream first to validate it against the spec. Thus the backflip to fully reading the CLks as
                 #       json into memory. -> issue #184
 
-                receipt_token, raw_file = upload_json_clk_data(dp_id, get_json(), span)
+                # get flag use_blocking from table projects
+                with DBConn() as conn:
```

Comment:
every time you call `with DBConn() as conn:` it will request a database connection. This creates some overhead. Thus it is better to group database interactions. About 20 lines above, we already had a database connection to do some queries. You could use that one instead of requesting a new one here.